hazelcast/src/test/java/com/hazelcast/map/MapPutAllWithBouncingMemberTest.java

```diff
@@ -57,11 +56,6 @@ public void setUp() {
         config = getConfig();
     }
 
-    @After
-    public void tearDown() {
-        factory.terminateAll();
-    }
```

Comment:
In this case `factory` is completely redundant, because `BounceMemberRule` has its own factory. In other cases, `factory` is closed by `HazelcastTestSupport.shutdownNodeFactory()` so close is also redundant. In addition, `factory` is needed for `assertUsedTieredStorage` so it should not be closed too early.

Reply:
btw, `testPutAll_whenAddingAndTerminatingMembers_thenPutAllShouldNotFail_withBatching` ignores "hazelcast.map.put.all.batch.size" setting...